fram

```diff
@@ -99,14 +153,21 @@ if [ "$ADMIN" == "true" ]; then
 	shell)
 		exec /bin/bash -l
 		;;
+	release)
+		rm $LOCKFILE
+		exit 0
+		;;
+	steal)
+		take_lock
+		exit 0
+		;;
 	esac
 fi
 
-# FIXME board locking/reservation
-
 # User commands
 case "$SSH_ORIGINAL_COMMAND" in
 	acc*)
+		check_lock
```

Comment:
That sounds good / simpler too, except handling 'unknown' commands might get tricky. It would have to fall through the first unlocked case, and then to fall to the 'unhandled command' case would have to pass the lock ...